src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/ModRewriteRedirectAction.cs

```diff
@@ -64,7 +64,7 @@ public override RuleResult ApplyAction(RewriteContext context, MatchResults rule
                     context.HttpContext.Response.Headers[HeaderNames.Location] = pattern + context.HttpContext.Request.QueryString;
                 }
             }
-            return RuleResult.ResponseComplete;
+            return RuleTermination.ResponseComplete;
```

Comment:
I don't like propagating the updating of the context here. It creates the notion that the action will always update the context, and is required, rather than the rule setting it. Ex RedirectToHttps doesn't have an action and sets the rewrite context directly.